modules/gapi/misc/python/pyopencv_gapi.hpp

```diff
@@ -346,22 +344,11 @@ static PyObject* extract_proto_args(PyObject* py_args, PyObject* kw)
         }
         else
         {
-            PyErr_SetString(PyExc_TypeError, "Unsupported type for cv.GIn()/cv.GOut()");
-            return NULL;
+            util::throw_error(std::logic_error("Unsupported type for GProtoArgs"));
```

Comment:
Because caller function handles this exception and setup `PyErr_SetString`.
https://github.com/TolyaTalamanov/opencv/blob/at/extend-python-bindings/modules/gapi/misc/python/pyopencv_gapi.hpp#L929